### PR TITLE
Update display after address has been edited

### PR DESCRIPTION
**Pull request by chibujax**
# JIRA TICKET NAME:
PDMO-59: [Update display after address has been edited](https://issues.openmrs.org/browse/PDMO-59)

## SUMMARY:
Update display after address has been edited

### CODE REVIEW

File: app/css/obsadmin.css [hunk 1]
```diff
@@ -426,6 +426,12 @@ li#location.selected a{
   margin-top: -10px;
 }
 
+.badge-no-color {
+  background-color: #f7f7f9;
+  color: #f7f7f9;
+}
+
+
 .badge-error {
```
Comment (jacksono) [inline app/css/obsadmin.css]: There seems to be an un necessary extra blank line here.

**Review by jacksono — COMMENTED** [review]
Code looks good.

**Comment by chibujax** [comment]
Thank you, @jacksono cleared.

**Comment by Ewanjiru** [comment]
Looks good

**Pull request merged**